tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -437,10 +510,17 @@ public void SetMultipleIndexer(int index, string sIndex, int value)
 			{
 				this[index, sIndex] = value;
 			}
+
+			public int GetMultipleIndexer(int index, string sIndex)
+			{
+				return this[index, sIndex];
+			}
 		}
 
 		public interface Fooish
 		{
+			int VirtualGet { get; set; }
```

Comment:
†) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.